chainermn/functions/__init__.py

```diff
@@ -0,0 +1,4 @@
+from chainermn.functions.point_to_point_communication import recv  # NOQA
+from chainermn.functions.point_to_point_communication import Recv  # NOQA
+from chainermn.functions.point_to_point_communication import send  # NOQA
+from chainermn.functions.point_to_point_communication import Send  # NOQA
```

Comment:
At the moment I do not feel they should be exposed, so I hid them in the latest commit.